src/pybind/mgr/cephadm/module.py

```diff
@@ -258,6 +259,13 @@ class CephadmOrchestrator(orchestrator.Orchestrator, MgrModule):
             'default': '/etc/prometheus/ceph/ceph_default_alerts.yml',
             'desc': 'location of alerts to include in prometheus deployments',
         },
+        {
+            'name': 'migration_current',
+            'type': 'int',
+            'default': None,
```

Comment:
`None`is not an integer, thus it will break code that relies on the configured type. It would be better to choose a value of `int`, e.g. `-1` or `0` here.
The regression introduced by this PR is https://tracker.ceph.com/issues/45963.